Aptoide/aptoide-client-v8
# [ASV-324] Metrics for install dialog on non logged in users

**What does this PR do?**

   Changed the analytics for the not logged in install dialog to identify the events that are related to the alternative flow

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] NotLoggedInShareAnalytics.java

**How should this be manually tested?**

  If you want you can set breakpoints when the event creator methods are called to check if the map has the alternative flow flag

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-324](https://aptoide.atlassian.net/browse/ASV-324)

**Questions:**


**Code Review Checklist**
